### ISSUE TITLE
youtube-dl downloads have become really slow due to throttling

### ISSUE THREAD

**Issue by XXLuigiMario** [bibanon/tubeup#174]
Since a couple of months ago, YouTube has begun throttling downloads for youtube-dl users to really slow speeds, ~75-80kbps. This makes downloads of videos take forever, especially for longer videos.

Currently this remains unfixed in youtube-dl. What's more, there are signs that its development has stopped, the last commit was on July 1st. The common workaround at this time is to switch to a more up to date fork, such as yt-dlp, which does have a fix in place.

**Comment by vxbinaca** [comment]
I don't disagree, youtube-dl needs to be replaced. Let me do a lib swap tomorrow and I'll push a new version that closes this.

**Comment by vxbinaca** [comment]
New version pushed.

**Comment by brandongalbraith** [comment]
@vxbinaca Looks like verbose is spitting out the same data as we're writing to <artifact identifier>.info.json. Thoughts on not being verbose on the command line?

**Comment by vxbinaca** [comment]
> @vxbinaca Looks like verbose is spitting out the same data to stdout as we're writing to {artifact_identifier}.info.json. Thoughts on not being verbose on the command line?

Not quote it censors the file location when writing it to disk. Also I talked to yt-dlps maintainer, he says we can filter out the video file URL and we'd likely be okay and I tend to agree. If it's possible to do that, just do it. The video location changes on every access and isn't fixed like a videoID in a URL.

**Comment by pukkandan** [comment]
The json is being printed to stdout due to this:

https://github.com/bibanon/tubeup/blob/72f12ad6acb627863f548f2cf0eec9b91d91e209/tubeup/TubeUp.py#L238

If you were using that in youtube-dl, it should have done the same too.

**Comment by brandongalbraith** [comment]
@pukkandan Yep, just "feels" like the JSON being dumped is more verbose than ytdl was ¯\_(ツ)_/¯. I should probably refactor so that output is suppressed when `--verbose` is not set.

**Comment by pukkandan** [comment]
> Yep, just "feels" like the JSON being dumped is more verbose than ytdl was

That may be possible. While I don't remember specifically adding more data to it, I had tweaked the related code. So it is possible I accidentally added some fields

**Comment by brandongalbraith** [comment]
@pukkandan No worries at all, and thank you for yt-dlp!

**Comment by vxbinaca** [comment]
Verbose in yt-dlp appears to behave differently than yt-dl. I mean if it doesn't effect metadata gathering I can clip the flag in the next version.
